api.planx.uk/modules/send/bops/bops.test.ts

```diff
@@ -119,3 +125,102 @@ describe(`sending an application to BOPS`, () => {
       });
   });
 });
+
+describe(`sending an application to BOPS v2`, () => {
+  beforeEach(() => {
+    queryMock.mockQuery({
+      name: "FindApplication",
+      data: {
+        bopsApplications: [],
+      },
+      variables: {
+        session_id: "123",
+        search_string: "%/api/v2/planning_applications",
+      },
+    });
+
+    queryMock.mockQuery({
+      name: "CreateBopsApplication",
+      matchOnVariables: false,
+      data: {
+        insertBopsApplication: { id: 22 },
+      },
+    });
+  });
+
+  it.todo("throws an error if payload is invalid");
```

Comment:
Let me know if you want another pair of eyes on this one, happy to jump on the branch and have a look.